camelot/view/controls/editors/datetimeeditor.py

```diff
@@ -129,18 +129,19 @@ def get_value(self):
             value = datetime.datetime(hour=time_value.hour(),
```

Comment:
een datetime editor w denk ik nergens gebruikt, enkel de delegate w gebruikt in de audit trail, dus die editor mag wat mij betreft ook gewoon weg, als dat eenvoudiger is